static-metric/Cargo.toml

```diff
@@ -23,7 +23,7 @@ quote = "1.0"
 lazy_static = "1.4"
 
 [dev-dependencies]
-prometheus = { version = "0.9", path = "../" }
+prometheus = { version = "0.10.0-rc.1", path = "../" }
```

Comment:
Let me know in case you would like me to bump the `static-metrics` version to `0.5.0` in this pull request as well, as a preparation for a future release.